pkgs/shells/nushell/plugins/hcl.nix

```
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  rustPlatform,
+  pkg-config,
+  nix-update-script,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nushell_plugin_hcl";
+  version = "0.104.1";
+
+  src = fetchFromGitHub {
+    repo = "nu_plugin_hcl";
+    owner = "Yethal";
+    rev = version;
```

```suggestion
    tag = version;
```
This is the preferred way to fetch tagged releases, it avoids accidentally fetching a branch of the same name.